build/commands/lib/pylint.js

```diff
@@ -48,7 +49,7 @@ const ensureDepotToolsInPath = (options) => {
 
 const getDefaultOptions = () => {
   let options = Object.assign({}, config.defaultOptions)
-  options.cwd = path.resolve(__dirname, '../../..')
+  options.cwd = path.join(config.rootDir, 'src/brave')
```

Comment:
same as above this should ideally be `config.braveCoreDir`
Sorry I missed that on the first pass, but would be great if you updated them